Game/SexEngine/FetishHolder.gd

```diff
@@ -33,6 +33,15 @@ func setCharacter(newchar):
 func setFetish(fetishID, interest):
 	fetishMap[fetishID] = interest
 
+func modifyFetishByTier(fetishID, numTiers: int = 1) -> bool:
+	var currentInterestValueNum = FetishInterest.interestToNumber(getFetishInterest(fetishID))
+	var projectedInterestValueNum = int(clamp((currentInterestValueNum + numTiers), -4, 4))
```

Comment:
I don't like the -4 and 4. Too hardcoded, might break. Best way to solve that is to add a clampInterestValue or something to FetishInterest class. It would still be hardcoded but at least its all would be in one file

Reply:
You mean the current system from "Hates" to "Loves" with 9 states could change?

Reply:
avoiding hardcoded numbers is clean code. The answer is yes, it could change at any moment